lib/models/project.js

```diff
@@ -104,6 +104,10 @@ Project.prototype.hasDependencies = function() {
  */
 Project.prototype.setupNodeModulesPath = function() {
   this.nodeModulesPath = nodeModulesPath(this.root);
+
+  this.emberCliNodeModulesPath = this.nodeModulesPath &&
```

Comment:
@stefanpenner @rwjblue Is getting ember-cli's `node_modules` path like this combined with `path.join(this.emberCliNodeModulesPath, 'ember-cli-legacy-blueprints')` below okay? 

It works when actually using it because the path exists, but in test, the path isn't valid so it doesn't import the addon, which is what is failing this test (because it can't find the component blueprint in test: ```  1 failing
  1) Acceptance: smoke-test ember can override and reuse the built-in blueprints:
     Error: the object {
  "code": 1
  "errors": [
    "Unknown blueprint: component\n"
  ]
  "output": [
    "DEPRECATION: Node v0.10.43 is no longer supported by Ember CLI. Please update to a more recent version of Node\n"
    "version: 2.4.2--6baf91e1a8\n"
    "Could not start watchman; falling back to NodeWatcher for file system events.\n"
    "Visit http://ember-cli.com/user-guide/#watchman for more info.\n"
    "installing component\n"
  ]
  "signal": [null]
} was thrown, throw an Error :)
      at lib$rsvp$$internal$$tryCatch (node_modules/rsvp/dist/rsvp.js:1036:16)
      at lib$rsvp$$internal$$invokeCallback (node_modules/rsvp/dist/rsvp.js:1048:17)
      at lib$rsvp$$internal$$publish (node_modules/rsvp/dist/rsvp.js:1019:11)
      at lib$rsvp$$internal$$publishRejection (node_modules/rsvp/dist/rsvp.js:962:7)
      at Object.lib$rsvp$asap$$flush [as _onImmediate] (node_modules/rsvp/dist/rsvp.js:1198:9)```